stubs/requests/requests/_internal_utils.pyi

```diff
@@ -0,0 +1,7 @@
+from re import Pattern
+from typing import Any
+
+HEADER_VALIDATORS: dict[type, tuple[Pattern[Any], Pattern[Any]]]
+
+def to_native_string(string: str | bytes, encoding: str = ...) -> str: ...
+def unicode_is_ascii(u_string: str) -> bool: ...
```

Comment:
I'm not really sure we should be including any stubs for a module that's named `_internal_utils`. Does stubtest complain if we omit it and `ignore_missing_stub` is set to `False`?

Reply:
It was locally, I can't speak to in CI/CD.

I'm torn on this though. If you want to work towards having upstream maintainers take over types, avoiding internal utility methods as have been done so far hurts that as they're critical to how the libraries work, in many cases.

Reply:
If upstream maintainers want to integrate typeshed's stubs into their codebases, that's great news! We're happy to help maintainers do that, and have done in the past. That's _not_ the primary purpose of the stubs packages typeshed provides, however.

Reply:
But what's the end goal of that? If there is no end game, doesn't that simply mean this is going to infinitely grow with no way to maintain it at scale, assuming appropriate interest. As such, I would argue that it's critical to take maintainers as a core customer not as secondary, because if they don't adopt stubs due to errors and whatnot, you end up in a lose-lose where they leave the work to you and you aren't prioritizing the work for them.

Just feels like this is being made harder on yourselves than you need to be because of these types of decisions, which in my opinion hurt the availability and use of types in upstream libraries